core/templates/dev/head/components/profile_link.html

```diff
@@ -0,0 +1,15 @@
+<script type="text/ng-template" id="components/profileLinkText">
+  <a ng-if="isUsernameLinkable(username)" ng-href="/profile/<[username]>">
+    <[username]>
+  </a>
+  <span ng-if="!isUsernameLinkable(username)">
+    <[username]>
+  </span>
+</script>
+
+<script type="text/ng-template" id="components/profileLinkImage">
+  <a ng-if="isUsernameLinkable(username)" ng-href="/profile/<[username]>">
+    <img ng-src="<[profilePicture]>" alt=""/>
```

Comment:
No need closing '/' on img tag -- it's a void tag. See http://stackoverflow.com/a/7854998